sormas-ui/src/main/java/de/symeda/sormas/ui/externalmessage/labmessage/LabMessageProcessingFlow.java

```diff
@@ -261,7 +261,13 @@ protected void handleCreateEventParticipant(
 		HandlerCallback<EventParticipantDto> callback) {
 		Window window = VaadinUiUtil.createPopupWindow();
 
-		EventParticipantEditForm createForm = new EventParticipantEditForm(event, false, eventParticipant.getPerson().isPseudonymized(), false);
+		EventParticipantEditForm createForm = new EventParticipantEditForm(
+			event,
+			false,
+			true,
+			eventParticipant.getPerson().isPseudonymized(),
```

Comment:
This ordering looks odd, until now, it jurisdiction flag came always after pseudonomization, now there is a bool inbetween

Reply:
It still comes after the pseudonymized flag, but there are two different pseudonymization flags in this method (one for the event participant, one for its person) and I actually mixed something up here. Good catch, fixed it.